Makefile

```diff
@@ -124,16 +117,18 @@ update-pot:
 install-po-files:
 	$(MAKE) -C po install RPM_BUILD_ROOT=$(DESTDIR)
 
-test:
-	@echo "***Running pylint$(PYVERSION) checks***"
-	@find . -name '*.py' -print|xargs -n1 --max-procs=$(NUM_PROCS) pylint$(PYVERSION) -E 2> /dev/null
-	@echo "[ OK ]"
-	@echo "***Running unittests checks***"
-	@PYTHONPATH=. py.test$(PYVERSION) --processes=-1 -vw tests/
```

Comment:
I wasn't able to find this tool.

Reply:
That's OK, `python -m pytest` does the same and even doesn't need the `PYTHONPATH` set.